src/pages/index.js

```diff
@@ -9,49 +9,90 @@ import Card from "../components/card.js";
 import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
```

Comment:
```js

  setEventListeners() {
    super.setEventListeners();
  }
```

Please, delete `setEventListeners`  from `PopupWithImage`

You shouldn’t create the **same methods** of the parent class if you don’t change it inside and only call it with `super`. The parent methods are available in any inheritor of the class.